RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Stream Engine
 
+# Release 1.3.8.1 2017-11-01
```

Comment:
1.3.8 is patch 8, there are no sub-patches in our current versioning. 
Please use 1.3.9.